src/test/scala/beam/router/GHRouterSpec.scala

```diff
@@ -29,25 +40,80 @@ class GHRouterSpec extends AnyWordSpecLike with Matchers with BeamHelper {
     .withFallback(config)
     .resolve()
 
-  lazy implicit val system: ActorSystem = ActorSystem("GHRouterSpec", config)
-
   "Static GH" must {
     "run successfully" in {
       runBeamWithConfig(config)
     }
 
     "add alternative route for GraphHopper if enabled" in {
-      val (matsimConfig, _, _) = runBeamWithConfig(configAltRoutes)
-      val outputDir = matsimConfig.controler.getOutputDirectory
-      val routingResponse = new File(outputDir, "ITERS/it.0/0.routingResponse.parquet")
-      val (iterator, closable) = ParquetReader.read(routingResponse.toString)
-      val result = iterator.exists { record =>
-        val router = record.get("router").toString
-        val itineraryIndex = record.get("itineraryIndex").toString
-        router == "GH" && itineraryIndex == "2"
-      }
-      closable.close()
-      result shouldBe true
+      lazy implicit val system: ActorSystem = ActorSystem("GHRouterSpec", configAltRoutes)
+      val (_, _, beamScenario: BeamScenario, services: BeamServices, _) = prepareBeamService(configAltRoutes, None)
+      val transportNetwork = services.injector.getInstance(classOf[TransportNetwork])
+
+      val request = RoutingRequest(
+        originUTM = new Location(167138.0, 2234.0),
+        destinationUTM = new Location(166321.9, 1568.87),
+        departureTime = 36952,
+        withTransit = false,
+        personId = Some(Id.createPersonId(2)),
+        streetVehicles = Vector(
+          StreetVehicle(
+            Id.createVehicleId(2),
+            Id.create("beamVilleCar", classOf[BeamVehicleType]),
+            SpaceTime(167138.00001152378, 2233.9999999985166, 32482),
+            BeamMode.CAR,
+            true,
+            true
+          ),
+          StreetVehicle(
+            Id.createVehicleId("body-2"),
+            Id.create("BODY-TYPE-DEFAULT", classOf[BeamVehicleType]),
+            SpaceTime(167138.0, 2234.0, 36952),
+            WALK,
+            true,
+            false
```

Comment:
For better readability could you add names to the params that are not clear from the context?

Reply:
done